ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -511,7 +511,7 @@ func getSubmitterTemplate(ctx context.Context, rayJobInstance *rayv1.RayJob, ray
 			return corev1.PodTemplateSpec{}, err
 		}
 		submitterTemplate.Spec.Containers[utils.RayContainerIndex].Command = []string{"/bin/bash"}
-		submitterTemplate.Spec.Containers[utils.RayContainerIndex].Args = []string{"-c", strings.Join(k8sJobCommand, " ")}
+		submitterTemplate.Spec.Containers[utils.RayContainerIndex].Args = []string{"-ce", strings.Join(k8sJobCommand, " ")}
```

Comment:
Add the `e` option to make the bash script exit when a command fails.

Reply:
Without the `-e` option, the bash script will not stop when the `ray job submit --no-wait` command fails. The script will then continue to execute `ray job logs` and fail again since the submission did not succeed. The result logs will be messed up because they now contain two failure messages, one for the `ray job submit --no-wait` and one for the `ray job logs`.

Reply:
Can you add a comment like "Without the -e option, the Bash script will continue executing even if a command returns a non-zero exit code."?

Reply:
Added.